evcouplings/compare/protocol.py

```diff
@@ -221,6 +226,168 @@ def _discrete_count(x):
+
+            # create the contact map
+            pairs.complex_contact_map(
+                ecs_i, ecs_j, ecs_inter,
+                d_intra_i, d_multimer_i,
+                d_intra_j, d_multimer_j,
+                d_inter,
+                margin=5,
+                boundaries=kwargs["boundaries"],
+                scale_sizes=kwargs["scale_sizes"]
+            )
+
+            # Add title to the plot
+            if ecs_inter is None:
+                ec_len = 'None'
+            else:
+                ec_len = len(ecs_inter)
+            plt.suptitle("{} inter-molecule evolutionary couplings".format(ec_len), fontsize=14)
+
+            # save to output
+            if output_file is not None:
+                plt.savefig(output_file, bbox_inches="tight")
+                plt.close(fig)
+
+    check_required(
+        kwargs,
+        [
+            "prefix", "min_sequence_distance",
+            "plot_probability_cutoffs",
+            "boundaries",
+            "draw_secondary_structure", "plot_lowest_count",
+            "plot_highest_count", "plot_increase",
+            "scale_sizes"
+        ]
+    )
+
+    prefix = kwargs["prefix"]
+
+    cm_files = []
+
+    ecs_longrange = ec_table.query(
+        "abs(i - j) >= {} or segment_i != segment_j".format(kwargs["min_sequence_distance"])
+    )
+
+    # create plots based on significance cutoff
+    if kwargs["plot_probability_cutoffs"]:
+        cutoffs = kwargs["plot_probability_cutoffs"]
+        if not isinstance(cutoffs, list):
+            cutoffs = [cutoffs]
+
+        for c in cutoffs:
+            ec_set = ecs_longrange.query("probability >= @c")
+
+            # only can plot if we have any significant ECs above threshold
+            if len(ec_set) > 0:
+                ec_set_i = ec_set.query("segment_i == segment_j == @first_segment_name")
+                ec_set_j = ec_set.query("segment_i == segment_j == @second_segment_name")
+                ec_set_inter = ec_set.query("segment_i != segment_j")
+
+                output_file = prefix + "_significant_ECs_{}.pdf".format(c)
+                plot_complex_cm(
+                    ec_set_i, ec_set_j, ec_set_inter,
+                    first_segment_name, second_segment_name,
+                    output_file=output_file
+                )
+                cm_files.append(output_file)
+
+    # range of plots to make
+    lowest = int(kwargs["plot_lowest_count"])
+    highest = int(kwargs["plot_highest_count"])
+    step = int(kwargs["plot_increase"])
+
+    for c in range(lowest, highest + 1, step):
+
+        # get the inter ECs to plot
+        ec_set_inter = ecs_longrange.query("segment_i != segment_j")[0:c]
+
+        # get the index of the lowest inter EC
+        last_inter_index = ec_set_inter.index[-1]
+
+        # take all intra-protein ECs that score higher than the lowest plotted inter-protein EC
+        ec_set_i = ecs_longrange.ix[0:last_inter_index].query("segment_i == segment_j == @first_segment_name")
```

Comment:
wrap for better readability, also don't use .ix which is deprecated (use .iloc instead), same for j

```
ec_set_i = ecs_longrange.iloc[0:last_inter_index].query(
    "segment_i == segment_j == @first_segment_name"
)
```